### ISSUE TITLE
[design] working memory / permanent memory = basically a user

### ISSUE THREAD

**Issue by imgntn** [imgntn/persist.io#58]
basically, the inventory for other people's patterns (you down wit' OPP!?)

working memory: session store of acquisitions.  cleared on exit.  patterns may be transferred from working to permanent memory by being attached to your garden.

block 0 | block 1 | block 2 | block 3 | block ...

permanent memory: attached to your garden, your initial pattern is block 0. block 0 is protected.  other blocks have decay rates after exit that are inversely proportional to their distance from block 0

block 0 | block 1 | block 2 | block 3 | block ...

basically a user is this series of patterns.
